src/rpcgovernance.cpp

```diff
@@ -846,6 +846,7 @@ UniValue getgovernanceinfo(const UniValue& params, bool fHelp)
             "{\n"
             "  \"governanceminquorum\": xxxxx,           (numeric) the absolute minimum number of votes needed to trigger a governance action\n"
             "  \"masternodewatchdogmaxseconds\": xxxxx,  (numeric) sentinel watchdog expiration time in seconds\n"
+            "  \"hassentinelping\": xxxxx,               (boolean) has sentinel ping rpc call\n"
```

Comment:
Why not ?  

"getgovernanceinfo's purpose is to print out global governance constants, not the state of the current masternode." - I don't know that it's purpose should be that narrowly construed.  It contains lastsuperblock and nextsuperblock which are not global constants.

Also hassentinelping isn't really about the state of the masternode but rather is a version dependent property.